AP-protocol.md

```diff
@@ -3,38 +3,77 @@
 
-The Network Initialization File has the following format:
-
-- 10 drone lines: `drone_id connected_drone_count connected_drone_ids packet_drop_rate`.
-
-    Consider this example line: `d1 3 d3 d4 d5 0.05`. It means that drone `d1` is connected with 3 drones `d3`, `d4` and `d5`, and that its Packet Drop Rate is 0.05 .
-
-    Note that the PDR is defined between 0 and 1 (0.05 = 5%).
-
-    Note that `connected_drone_ids` cannot contain `drone_id` nor repetitions.
-
-- 2 client lines: `client_id connected_drone_count connected_drone_ids`.
-
-    Consider this example line: `c1 2 d2 d3`. It means that client `c1` is connected with 2 drones `d2` and `d3`.
-
-    Note that `connected_drone_ids` cannot contain `client_id` nor repetitions.
+> Importantly, the Network Initializer should also setup the Rust channels between the nodes and the Simulation Controller (see the Simulation Controller section).
 
-    Note that a client cannot connect to other clients or servers.
+## Network Initialization File
+The **Network Initialization File** is in the `.toml` format, and structured as explained below:
 
-- 2 server lines: `server_id connected_drone_count connected_drone_ids`.
-
-    Consider this example line: `s1 2 d4 d5`. It means that server `s1` is connected with 2 drones `d4` and `d5`.
-
-    Note that `connected_drone_ids` cannot contain `server_id` nor repetitions.
-
-    Note that a server cannot connect to other clients or servers.
-
-Importantly, the Network Initializer should also setup the Rust channels between the nodes and the Simulation Controller (see the Simulation Controller section).
+### Drones
+Any number of drones, each formatted as:
+```TOML
+[[drone]]  
+id = "drone_id"  
+connected_drone_ids = ["connected_id1", "connected_id2", "connected_id3", "..."]  
+pdr = "pdr"
+```
+- note that the `pdr` is defined between 0 and 1 (0.05 = 5%).
+- note that `connected_drone_ids` cannot contain `drone_id` nor repetitions
+
+### Clients
+Any number of clients, each formatted as:
+```TOML
+[[client]]  
+id = "client_id"  
+connected_drone_ids = ["connected_id1", "..."] # max 2 entries
+```
+- note that `connected_drone_ids` cannot contain `client_id` nor repetitions
+- note that a client cannot connect to other clients or servers
+- note that a client can be connected to at most two drones
+
+### Servers
+Any number of servers, each formatted as:
+```TOML
+[[server]]  
+id = "server_id"  
+connected_drone_ids = ["connected_id1", "connected_id2", "connected_id3", "..."] # at least 2 entries
+```
+- note that `connected_drone_ids` cannot contain `server_id` nor repetitions
+- note that a server cannot connect to other clients or servers
+- note that a server should be connected to at least two drones
+
+### Example
+```toml
+[[drone]]  
+id = "d1"  
```

Comment:
The only thing that should be changed is that we are using elsewhere u64 for client and server (and having them as a string is stupid)

Reply:
Thanks, I didn't notice I had left them in as strings, fixed in df61622